src/installation/guides/zfs.md

```diff
@@ -0,0 +1,271 @@
+# ZFS On Root
+
+This installation guide assumes that an already existing Full Void Linux install
+is available. Not just a Live USB.
+
+If an existing Void install does not exist please consult one of the [other
+installation guides](./index.md) or use a custom iso such as
+[hrmpf](https://github.com/leahneukirchen/hrmpf) then continue along with this
+guide.
+
+## Setup
+
+From the permanent system, install the `zfs` package.
+
+Then ensure the ZFS module is loaded with
+[modprobe(1)](https://man.voidlinux.org/modprobe):
+
+`# modprobe zfs`
+
+Locate the disk to format using [fdisk(8)](https://man.voidlinux.org/fdisk) or
+[lsblk(8)](https://man.voidlinux.org/lsblk):
+
+`# fdisk -l`
+
+## Partitioning
+
+Prepare the drive for installation by creating one of the following partition
+schemes using [cfdisk(8)](https://man.voidlinux.org/cfdisk) or another
+partitioning tool:
+
+`# cfdisk -z /dev/sda`
+
+> **Warning:**
+> 
+> The disk being partitioned will be formatted and any existing data on the disk
+> will be destroyed.
+
+For a BIOS/MBR system:
+
+```
+Partition    Size       Type
+1             ?G     Solaris Root(bf00)
+```
+
+For a BIOS/GPT system:
+
+```
+Partition    Size       Type
+1             2M     BIOS boot    
+2             ?G     Solaris Root
+```
+
+ZFS pools should use id's to maintain universal functionality across computers
+and to prevent the misrecognition of drives.
+
+Id's can be located in the `/dev/disk/by-id` folder:
+
+> **Note:**
+> 
+> `wwn-` entries and `ata-` entries are referencing equivalent drvies and either
+> can be used.
+
+```
+$ ls -l /dev/disk/by-id
+total 0
+lrwxrwxrwx 1 root root  9 Apr 16 17:34 ata-Hitachi_HDS721010CLA332_JP6911HD2AHS9F -> ../../sda
+lrwxrwxrwx 1 root root 10 Apr 16 17:34 ata-Hitachi_HDS721010CLA332_JP6911HD2AHS9F-part1 -> ../../sda1
+lrwxrwxrwx 1 root root 10 Apr 16 17:34 ata-Hitachi_HDS721010CLA332_JP6911HD2AHS9F-part2 -> ../../sda2
+lrwxrwxrwx 1 root root  9 Apr 16 17:34 wwn-0x5000cca373e0f5d9 -> ../../sda
+lrwxrwxrwx 1 root root 10 Apr 16 17:34 wwn-0x5000cca373e0f5d9-part1 -> ../../sda1
+lrwxrwxrwx 1 root root 10 Apr 16 17:34 wwn-0x5000cca373e0f5d9-part2 -> ../../sda2
+```
+
+Create a pool specifying the previously created Solaris Root Partition from the
+disk:
+
+> **Note:**
+> 
+> Many users with Advanced Format hard drives will want to also add the `-o
+> ahisft=12` argument. This sets the drive's logical sectors to 4k in size.
+
+`# zpool create -f -m none zroot dev`
+
+| Command | Action                                            |
+|---------|---------------------------------------------------|
+| -f      | Force the creation of the pool                    |
+| -m none | Set the mountpoint to none                        |
+| zroot   | The name of the pool                              |
+| dev     | The device to be used in the creation of the pool |
+|         | (ie. ata-XXXXX-partX or wwn-XXXX-partX)           |
```

Comment:
I'm not entirely sure we should use this table, but if it were to stay I would rename the "Command" column to "Argument" and use code blocks for all of them.